apps/src/applab/designElements/photochooser.jsx

```diff
@@ -0,0 +1,226 @@
+        <PropertyRow
+          desc={'id'}
+          initialValue={elementUtils.getId(element)}
+          handleChange={this.props.handleChange.bind(this, 'id')}
+          isIdRow
+        />
+        <PropertyRow
+          desc={'width (px)'}
+          isNumber
+          initialValue={parseInt(element.style.width, 10)}
+          handleChange={this.props.handleChange.bind(this, 'style-width')}
+        />
+        <PropertyRow
+          desc={'height (px)'}
+          isNumber
+          initialValue={parseInt(element.style.height, 10)}
+          handleChange={this.props.handleChange.bind(this, 'style-height')}
+        />
+        <PropertyRow
+          desc={'x position (px)'}
+          isNumber
+          initialValue={parseInt(element.style.left, 10)}
+          handleChange={this.props.handleChange.bind(this, 'left')}
+        />
+        <PropertyRow
+          desc={'y position (px)'}
+          isNumber
+          initialValue={parseInt(element.style.top, 10)}
+          handleChange={this.props.handleChange.bind(this, 'top')}
+        />
+        <ColorPickerPropertyRow
+          desc={'background color'}
+          initialValue={elementUtils.rgb2hex(element.style.backgroundColor)}
+          handleChange={this.props.handleChange.bind(this, 'backgroundColor')}
+        />
+        {iconColorPicker}
+        <BorderProperties
+          element={element}
+          handleBorderWidthChange={this.props.handleChange.bind(
+            this,
+            'borderWidth'
+          )}
+          handleBorderColorChange={this.props.handleChange.bind(
+            this,
+            'borderColor'
+          )}
+          handleBorderRadiusChange={this.props.handleChange.bind(
+            this,
+            'borderRadius'
+          )}
+        />
+        <BooleanPropertyRow
+          desc={'hidden'}
+          initialValue={$(element).hasClass('design-mode-hidden')}
+          handleChange={this.props.handleChange.bind(this, 'hidden')}
+        />
+        <ZOrderRow
+          element={this.props.element}
+          onDepthChange={this.props.onDepthChange}
+        />
+      </div>
+    );
+  }
+}
+
+class PhotoChooserEvents extends React.Component {
+  static propTypes = {
+    element: PropTypes.instanceOf(HTMLElement).isRequired,
+    handleChange: PropTypes.func.isRequired,
+    onInsertEvent: PropTypes.func.isRequired
+  };
+
+  getPhotoselectEventCode() {
+    const id = elementUtils.getId(this.props.element);
+    const code =
+      'onEvent("' +
+      id +
+      '", "change", function() {\n' +
+      '  console.log("' +
+      id +
+      ' photo selected!");\n' +
+      '  console.log(getImageURL("' +
+      id +
+      '"));\n' +
+      '});\n';
+    return code;
+  }
+
+  insertPhotoselect = () =>
+    this.props.onInsertEvent(this.getPhotoselectEventCode());
+
+  render() {
+    const element = this.props.element;
+    const clickName = 'Change';
+    const clickDesc = 'Triggered when a photo is selected.';
```

Comment:
NIT: Please add these to i18n so they can get translated :) 
Example: https://github.com/code-dot-org/code-dot-org/pull/32292/commits/72a98cbee4600f4b43b103718189e628bb25cc3f